src/main/java/calculator/Delimiter.java

```diff
@@ -0,0 +1,34 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Delimiter {
+
+    public static final String DEFAULT_DELIMITER = "[,:]";
+    public static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    public static final int DELIMITER_INDEX = 1;
+    public static final int NUMBER_INDEX = 2;
+
+    private String delimiter = DEFAULT_DELIMITER;
+
+    public List<String> split(String text) {
+        Matcher m = Pattern.compile(CUSTOM_DELIMITER).matcher(text);
+
+        if (m.find()) {
+            delimiter = m.group(DELIMITER_INDEX);
+            return Arrays.asList(m.group(NUMBER_INDEX).split(delimiter));
+        }
+        return Arrays.asList(text.split(delimiter));
+    }
+
+    public boolean isSplit(String text) {
+        if (text == null) {
+            return false;
+        }
+
+        return !text.isEmpty();
```

Comment:
```suggestion
        return text!=null && !text.isEmpty();
```
요렇게 간단히 표현할 수 있습니다 :) 
isSplit 보단 null인지를 체크하는 메서드 명이 좋을 것 같아요!